packages/calcite-components/src/components/slider/slider.scss

```diff
@@ -4,21 +4,19 @@
  * These properties can be overridden using the component's tag as selector.
  *
  * @prop --calcite-slider-fill-color: Specifies the color of the fill.
- * @prop --calcite-slider-handle-border-color-hover: Specifies the border color of the handle when hovered.
  * @prop --calcite-slider-handle-border-color: Specifies the border color of the handle.
+ * @prop --calcite-slider-handle-border-color-hover: Specifies the border color of the handle when hovered.
+ * @prop --calcite-slider-handle-color: Specifies the color of the handle.
  * @prop --calcite-slider-handle-color-active: Specifies the color of the handle.
  * @prop --calcite-slider-handle-color-hover: Specifies the color of the handle when hovered.
- * @prop --calcite-slider-handle-color: Specifies the color OF the handle.
- * @prop --calcite-slider-handle-shadow-hover: Specifies the shadow of the handle when hovered.
- * @prop --calcite-slider-handle-shadow: Specifies the shadow of the handle.
  * @prop --calcite-slider-histogram-accent-color: Specifies the accent color of the graph.
  * @prop --calcite-slider-histogram-background-color: Specifies the background color of the graph.
- * @prop --calcite-slider-precise-handle-color-hover: Specifies the color of the handle extension when hovered.
  * @prop --calcite-slider-precise-handle-color: Specifies the color of the handle extension.
+ * @prop --calcite-slider-precise-handle-color-hover: Specifies the color of the handle extension when hovered.
  * @prop --calcite-slider-text-color: Specifies the text color of the component.
- * @prop --calcite-slider-text-size: Specifies the text size of the component.
  * @prop --calcite-slider-tick-border-color: Specifies the border color of the tick.
  * @prop --calcite-slider-tick-color: Specifies the color of the tick.
+ * @prop --calcite-slider-tick-color-active: Specifies the color of the active tick.
  * @prop --calcite-slider-track-color: Specifies the color of the track.
```

Comment:
To sum these changes up:

1. Removed these unused tokens: 
- `--calcite-slider-handle-shadow-hover`
- `--calcite-slider-handle-shadow`
- `--calcite-slider-text-size`
2. Alpha sorted the remaining.